docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -157,6 +157,27 @@ If the battery is low, the connection between your station and the WG1000 gatewa
 1. Check if the station's battery level is low through mobile app settings.
 2. Check for [Helium coverage](https://docs.weatherxm.com/helium-bundle/helium-troubleshooting#how-to-see-if-there-is-helium-coverage-in-your-area) in your area.
 
+### For the WG1200 Gateway
+
+<div class="explanation">
+   <img src={useBaseUrl("/img/home/D1-icons.png")} />
+  <div>
+    <h4>WG1200 Gateway icons explanation <br/> 
+    Red=Problem, Green= Good</h4>
+    <ol>
+      <li>RF connection with the station.</li>
+      <li>Connection to WeatherXM Network.</li>
+      <li>GPS receiver.</li>
+      <li>WiFi connection.</li>
+    </ol>
+  </div>
+</div>
+
+- Check that your station connection (RF) to the gateway is stable by following the guide [here](/wxm-devices/D1-bundle/D1-troubleshooting#weather-station-not-reporting-data-to-wg1200-gateway-there-are-dashes----on-the-display-console).
+- If the station's battery level a warning will appear on the [WG1200 Gateway](/wxm-devices/D1-bundle/D1-troubleshooting#one-or-more-variables-are-extremely-high-low-or-constant-for-long-periods-of-time). 
+- Follow the Troubleshooting guide regarding [No WiFi connection](/wxm-devices/D1-bundle/D1-troubleshooting#no-wifi--connection).
```

Comment:
These are good, but how about GPS turning red?? Do we have something in that case to troubleshoot the user?

Reply:
I think the GPS is in the Gateway as M5, let's confirm it with the hardware team.